src/main/features/claim/views/eligibility/single-defendant.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+
+{% set title = t('Single Defendant | Money claim') %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Is this claim against only one individual or business?') }}</h1>
+      <p>{{ t('You can’t use this service if this claim is against more than one person or business.') }}</p>
+
+      {% set name = 'singleDefendant' %}
+
+      <form novalidate method="post" class="analytics-submit-event-trigger"
+            data-event-action="Eligibility: Select Yes"
+            data-event-label-from="{{ name }}">
+        {{ csrfProtection(csrf) }}
+
+        {{
+          radioGroup(
+            'Is this claim against only one individual or business?',
+            name,
+            [
+              {'label': 'Yes', value: YesNoOption.YES.option, checked: (form.valueFor(name) === YesNoOption.YES)},
+              {'label': 'No', value: YesNoOption.NO.option, checked: (form.valueFor(name) === YesNoOption.NO)}
+            ],
+            form = form,
+            labelForScreenReadersOnly = true
+          )
+        }}
+
```

Comment:
No need to have two blank lines.

Reply:
Removed but this comment will not update as it covers 1 extra line